Frameworks/include/UIPopoverPresentationControllerInternal.h

```diff
@@ -16,14 +16,11 @@
 
 #pragma once
 
-@interface UIPopoverPresentationController () {
-@public
-    BOOL _managesPresentation;
-    dispatch_block_t _willDismissCompletion;
-}
+@interface UIPopoverPresentationController()
```

Comment:
The file is named UIPopoverPresentationControllerInternal - generally the interfaces match that name as well.